src/main/java/ru/itis/services/WishListService.java

```diff
@@ -36,4 +37,13 @@ public boolean removeByTitle(String title, String token) {
         wishListCandidate.ifPresent(wishList -> wishListRepository.delete(wishList));
         return false;
     }
+
+    public Optional<WishList> findWishListByTitle(String title) {
+        return wishListRepository.findByTitle(title);
```

Comment:
Why do you need this class if it only delegates?

Reply:
some actions can not be made in controller->need of service